### ISSUE TITLE
New weapon: Blood Oath

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#255]
"Infinite ammo is cursed", they said. Perhaps they weren't wrong...

copy from AR or SMG
infinite spare ammo
buff stats to compensate for drawbacks

Special: reloading this gun deals 30 damage. Also while firing, you are constantly taking 0.01 damage per second, which prevents regen.

Hard to use gun, all about HP management.

**Comment by ghost** [comment]
Reloading does dmg to who? Enemies nearby, or you? 
*If*, reloading deals 30 damage to you, then I don't see the point of the constant damage, because it is an unnecessary nerf. The infinite ammo to reload to, is already balanced by the 30 damage.

**Comment by deferred-impact** [comment]
You take damage when reloading, yes.

I find the "no regen while firing" feature to be thematically fitting, and it also offers room for further balancing (this weapon disables all regen, sure, but we can make it stronger in other regards)

**Comment by ghost** [comment]
Yes, I would recommend buffing it in other regards, since it doesn't allow a user to heal.

**Comment by deferred-impact** [comment]
The main post did say:

> buff stats to compensate for drawbacks

**Comment by ghost** [comment]
Yeah
